starport-lambda/src/main/scala/com/krux/starport/lambda/SubmitHandler.scala

```diff
@@ -69,5 +79,12 @@ class SubmitHandler extends RequestHandler[SubmitRequest, SubmitResponse] with L
       .map(f => s"${f.getAbsolutePath}: ${f.length()}")
       .mkString("\n")
   }
-}
 
+  /**
+   * @return delete the /tmp directory
+   */
+  private def deleteTmpDir(): String = {
+    new File("/tmp").delete
```

Comment:
I thought we have a prefix? Might not be a good idea to delete every thing in `/tmp`

Reply:
There's no prefix in the `scanTmpFiles` method - I assumed we wanted the same directory.

Reply:
let's create a subfolder in `/tmp`

Reply:
OK this is done now.